frontend/src/test/bookings-group-exam.test.js

```diff
@@ -0,0 +1,187 @@
+const puppeteer = require("puppeteer");
+
+let page;
+let browser;
+const width = 1200;
+const height = 800;
+const maxTestCaseTime = 41000;
+
+beforeAll(async () => {
+    browser = await puppeteer.launch({
+        headless: false,
+        slowMo: 50,
+        args: [`--window-size=${width},${height}`]
+    });
+    page = await browser.newPage();
+    await page.setViewport({ width, height });
+});
+
+afterAll(() => {
+    browser.close();
+});
+
+describe("Serve Citizens", () => {
+  test(
+    "Login",
+    async () => {
+      await page.goto(process.env.CFMS_DEV_URL);
+      await page.waitForSelector("#keycloak-login");
+
+      const navigationPromise = page.waitForNavigation();
+      await page.click("#keycloak-login");
+      await navigationPromise;
+
+      await page.waitForSelector("#username");
+      await page.type("#username", "cfms-postman-operator");
+      await page.type("#password", process.env.POSTMAN_OPERATOR_PASSWORD);
+
+      await page.click("#kc-login");
+      await navigationPromise;
+
+      await page.waitForSelector("label.navbar-user");
+    },
+    maxTestCaseTime
+  );
+
+test(
+    "Navigate to Exam Inventory Page",
```

Comment:
This test seems poorly named. It looks like it's the "Add Group Exam" test?

Reply:
fixed